### ISSUE TITLE
Send metadata and custom attributes using the un-secure mode.

### ISSUE THREAD

**Issue by paologallinaharbur** [newrelic/newrelic-coreint#192]
We need a way to send custom attributes and metadata when using un-secure mode. 
This is needed since KSM  is going to leverage that as kube-events.

We can send custom attributes together with the payload including them at agent level.

Regarding the metadata how to do that is still an open question.

**Comment by paologallinaharbur** [comment]
UPDATE: this is not needed anymore for the control plane components

**Comment by paologallinaharbur** [comment]
- [ ] test secure/unsecure mode
 - [ ] check in infra code if it is possible to make the change and create a POC
 - [ ] agree with CAOS and Julien regarding how to include this change
 - [ ] agree with Roberto Julien regarding the possibility to drop the support for metadata decoration (since is currently half broken)

**Comment by kang-makes** [comment]
Blocked by newrelic/infrastructure-agent#911

**Comment by kang-makes** [comment]
After the stand up we agreed this is not blocked but a known issue. I have created another issue to to the follow up: newrelic/nri-kube-events#85

Closing.
